modules/programs/git.nix

```diff
@@ -76,6 +76,16 @@ in
         type = types.attrsOf types.attrs;
         internal = true;
       };
+
+      ignores = mkOption {
+        type = types.nullOr types.lines;
```

Comment:
I think it would make sense to have `type = types.listOf types.str` here. Then the default would be `[]` and the example `[ "*~" "*.swp" ]`.

Reply:
Hm, I did orientate on `programs.bash`, which uses `types.lines`, but I can of course change this to `types.listOf types.str`.